apps/condo/domains/common/utils/next/auth.tsx

```diff
@@ -0,0 +1,42 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
+import { AuthenticatedUserQuery, AuthenticatedUserDocument, AuthenticatedUserQueryResult } from '@app/condo/gql'
+import { GetServerSideProps, GetServerSidePropsContext } from 'next'
+
+
+type PrefetchAuthType = (client: ApolloClient<NormalizedCacheObject>) => Promise<AuthenticatedUserQueryResult['data']['authenticatedUser']>
+
+export const prefetchAuth: PrefetchAuthType = async (client) => {
```

Comment:
It's better to use `function`s when possible. const are resolved as anonymous promise, so you'll miss information about it in error trace, if something goes wrong

Reply:
fixed